packages/amplify-cli/src/domain/amplify-telemetry/getTelemetryUrl.ts

```diff
@@ -0,0 +1,10 @@
+import url, { UrlWithStringQuery } from 'url';
+import { getLatestApiVersion } from './VersionManager';
+
+const version = getLatestApiVersion();
+const prodUrl = `https://aws-amplify-cli-telemetry.us-east-1.amazonaws.com/${version}/metrics`;
+export function getUrl(): UrlWithStringQuery {
```

Comment:
Are we caching the result of this function between invocations?

Reply:
No we are not caching it. It's kind of excessive to cache something that's been called twice in a lifetime when the software is running